modules/claims_api/lib/claims_api/evss_bgs_mapper.rb

```diff
@@ -73,13 +73,25 @@ def claim_phase_dates
 
       obj = [@data['bnft_claim_lc_status']].flatten
       latest_phase_info = obj.max_by { |d| d['phase_chngd_dt'] }
-      phase_num = latest_phase_info['phase_type_change_ind'][0]
+      phases = phases(obj)
       {
         'latest_phase_type' => latest_phase_info['phase_type'],
         'phase_change_date' => format_bgs_date(latest_phase_info['phase_chngd_dt']),
-        'phase_type_change_ind' => latest_phase_info['phase_type_change_ind'],
-        "phase#{phase_num}_complete_date" => format_bgs_date(latest_phase_info['phase_chngd_dt'])
-      }
+        'phase_type_change_ind' => latest_phase_info['phase_type_change_ind']
+      }.merge(phases)
```

Comment:
Could this just be `.merge(phases(obj))`? I guess it's less clean, but takes 1 less line of code *shrug*